admin/class-yoast-form.php

```diff
@@ -545,4 +547,25 @@ public function toggle_switch( $var, $values, $label ) {
 
 		echo '<a></a></div></fieldset><div class="clear"></div></div>' . "\n\n";
 	}
+
+	/**
+	 * Check if the user is on a page of Yoast SEO
+	 *
+	 * @return bool
+	 */
+	protected function load_i18n_promo_class() {
+		$wpseo_admin = new WPSEO_Admin();
+
+		$current_screen = get_current_screen()->id;
```

Comment:
Unfortunately not. `is_yoast_seo_page` returns a boolean and also returns true for the addon pages.